### PR TITLE
Add dictionary definition for "edge case"

### PR DESCRIPTION
**Pull request by monicacodes**
## This PR fixes...

the lack of dictionary definition and perspective for "edge case"

## What I did...

- Add dictionary definition for edge case
- Add software engineer perspective for edge case

## How to test...

1. Navigate to /dictionary
1. Search "edge case"
1. See if definition and perspective shows up under "edge case"
2. Check grammar and typos

## I learned...

the dictionary definition of what an edge case is in software testing.

### CODE REVIEW

File: src/assets/content/dictionary/edge-case.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 title: edge case
-definition: ''
-perspectives: []
+definition: An edge case is a type of software bug that are uncommon for users to encounter.
+sources:
+- sourceurl: https://airfocus.com/glossary/what-is-an-edge-case/
+perspectives:
+- meaning: should not be overlooked or underestimated.
```
Comment (alliequintano) [inline src/assets/content/dictionary/edge-case.md]:
```suggestion
- meaning: a case that, while unusual, should still not be overlooked or underestimated
```

**Review by alliequintano — CHANGES_REQUESTED** [review]
Hi @monicacodes ! Thank you for your contribution. I've added a suggestion to change the wording of your perspective to make it read a bit easier. You can also see that a period is not needed at the end of the meaning because it will be added when the meaning is displayed in the perspective sentence. 

I tested your changes using the Netlify preview. See the screenshot below:
image

File: src/assets/content/dictionary/edge-case.md (hunk 1 above)
Comment (monicacodes) [reply src/assets/content/dictionary/edge-case.md]: Hi @alliequintano! Thanks for the suggestion. I committed your suggestion. Let me know if anything else needs to be changed.

**Review by novellac — APPROVED** [review]
I'll go ahead and accept this on @alliequintano 's behalf. Thanks and happy Hacktoberfest! 🍁
